packages/server/test/unit/browsers/firefox_spec.ts

```diff
@@ -126,7 +126,7 @@ describe('lib/browsers/firefox', () => {
 
   context('#open', () => {
     beforeEach(function () {
-      this.browser = { name: 'firefox', channel: 'stable' }
+      this.browser = { name: 'firefox', channel: 'stable', majorVersion: 100 }
```

Comment:
Why is `majorVersion` needed now?

Reply:
The tests I added need [`hasCdp` to be true](https://github.com/cypress-io/cypress/blob/becb893f07ba3f85f385cc69650e4db4a49e3b95/packages/server/lib/browsers/firefox.ts#L404) since the `after:browser:launch` event needs the remote debugging port.

Reply:
can we add an in-line comment about this version adding hasCDP? feels like something we'll ask about in the future and forget why it's here

or maybe there is a better place to add this info?

Reply:
Added a comment in https://github.com/cypress-io/cypress/pull/28180/commits/2d862affcacd5f810dbcc2c4e888e10173e1ddff